src/textual/dom.py

```diff
@@ -514,7 +514,7 @@ def get_component_styles(self, *names: str) -> RenderStyles:
         """Get a "component" styles object (must be defined in COMPONENT_CLASSES classvar).
 
         Args:
-            name: Name of the component.
+            names: Names of the components.
```

Comment:
This whole docstring for `get_component_styles` might need updating to reflect recent changes?

Reply:
Yep, noted.